ng2-components/ng2-alfresco-core/src/components/context-menu/context-menu-holder.component.ts

```diff
@@ -48,9 +48,13 @@ export class ContextMenuHolderComponent implements OnInit, OnDestroy {
     private openSubscription: Subscription;
     private closeSubscription: Subscription;
     private contextSubscription: Subscription;
+    private contextMenuListenerFn: () => void;
 
-    @Input() showIcons: boolean = false;
-    @ViewChild(MatMenuTrigger) menuTrigger: MatMenuTrigger;
+    @Input()
+    showIcons: boolean = false;
+
+    @ViewChild(MatMenuTrigger)
+    menuTrigger: MatMenuTrigger;
 
     @HostListener('contextmenu', ['$event'])
```

Comment:
is this still relevant since it's overplayed by material component?

Reply:
Yes, did smoke testing and should work fine.